### ISSUE TITLE
Cannot refresh .tfstate (Google Cloud)

### ISSUE THREAD

**Issue by phanect** [hashicorp/terraform#4539]
1. Remove Google Cloud SQL instance named "a" from **web interface**
2. Run `terraform refresh`
3. Following error is shown and cannot refresh .tfstate file

```
* google_sql_database_instance.master: Error retrieving instance a: googleapi: Error 404: The Cloud SQL instance does not exist., instanceDoesNotExist
```

3-expected. Information about Google Cloud SQL instance "a" is remove from .tfstate file

Terraform version: 0.6.8 and latest `master` f99565157393b7cb3c49377cd44b127b523dc85e
Platform: Kubuntu 14.04 x86_64 (Kernel 3.13.0-74)

**Comment by phinze** [comment]
Thanks for the report! We should definitely be catching that error and allowing Terraform to make progress.

Pinging @lwander this should be a quick fix methinks.

**Comment by lwander** [comment]
See #3913

**Comment by radeksimko** [comment]
Fixed via #3913

**Comment by phanect** [comment]
Thank you for rapid fix!
